python/tvm/relay/op/contrib/ethosu.py

```diff
@@ -915,6 +915,59 @@ def abs_pattern() -> tvm.relay.dataflow_pattern.DFPattern:
     return pattern
 
 
+class ConcatParams:
+    """
+    This class will parse a call to a ethos-u.concat composite function
+    and extract the parameter information.
+    """
+
+    composite_name = "ethos-u.concat"
+
+    def __init__(self, func_body):
+        self.concat = func_body
+        self.input_tensors = [TensorParams(tensor) for tensor in list(func_body.args[0])]
+        self.input_scales = [s.data.asnumpy() for s in list(func_body.args[1])]
+        self.input_zero_points = [zp.data.asnumpy() for zp in list(func_body.args[2])]
+        self.axis = func_body.attrs.axis
+
+    def is_valid(self):
+        """Checks whether Concatenate has compatible attributes with the hardware"""
+        if not check_valid_dtypes(self.input_tensors, supported_dtypes=[np.int8]):
```

Comment:
`uint8` should be supported as well, but I didn't enable it since we don't test it... It would be trivial to enable it in the future if we need to support TFLite 1